### DISCUSSION TITLE
Free joint input problem

### DISCUSSION THREAD

**Discussion by LHT-Liu** [google-deepmind/mujoco#2055] [Asking for Help]
### Intro

I am a student, I use MuJoCo for computing Jacobian matrix with free joint 

### My setup

Apple M1 Pro macOS Sonoma 14.6.1

PyCharm 2023.3.5 (Community Edition)

mujoco 3.2.2

### My question

for` free joint`
`joint_angles = [trans[0], trans[1], trans[2], quat.w, quat.x, quat.y, quat.z,......]`
Regarding the freee joint, is the order of the last four input parameters `quat.w, quat.x, quat.y, quat.z` correct?

For pinocchio, this sequence is reversed (quat.x, quat.y, quat.z, quat.w), right?

### Minimal model and/or code that explain my question

Thanks
@yuva

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
> Regarding the freee joint, is the order of the last four input parameters quat.w, quat.x, quat.y, quat.z correct?

Yes

> For pinocchio, this sequence is reversed (quat.x, quat.y, quat.z, quat.w), right?

I don't know, read their docs I guess.

**Reply by LHT-Liu** [reply]
Thanks
